backend/src/api/routes/advisor/index.routes.ts

```
@@ -0,0 +1,75 @@
+);
+
+/**
+ * Get findings for the latest advisor scan.
+ * GET /api/advisor/issues
+ */
+router.get(
+  '/issues',
+  verifyAdmin,
+  async (req: AuthRequest, res: Response, next: NextFunction) => {
+    try {
+      const severity = req.query.severity as string | undefined;
```

ESLint

[error] Delete `··`

(prettier/prettier)